pkgs/tools/system/retry/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, libtool, automake, autoconf, which, txt2man }:
+stdenv.mkDerivation rec {
+    pname = "retry";
+    version = "1.04";
+
+    buildInputs = [ libtool automake autoconf which txt2man ];
+
+    src = fetchFromGitHub {
+        owner = "minfrin";
+        repo = "retry";
+        rev = "f6f1acd4b235e96dd9cb11263fb181ec44c3ece9";
+        sha256 = "sha256:0jrx4yrwlf4fn3309kxraj7zgwk7gq6rz5ibswq3w3b3jfvxi8qb";
+    };
+
+    buildConfigScript = ''
+        libtoolize --force
+        aclocal
+        autoheader
+        automake --add-missing
+        autoconf
+    '';
```

Comment:
We can probably replace this with autoreconfHook.

Reply:
Replaced.